docs/Deployment/Production-Checklist.md

```diff
@@ -8,15 +8,17 @@ menuOrder: 2
 This is a checklist for configuring and securing Cube.js for a production
 deployment.
 
-## Enable Production Mode
+## Disable Dev Mode
```

Comment:
I think that, generally, we should refrain from using contractions because they decrease the accessibility of our docs. It's okay to have `DEV` in the code though.
```suggestion
## Disable Development Mode
```